Documentation/dev-guide/apispec/swagger/rpc.swagger.json

```diff
@@ -34,6 +34,12 @@
             "schema": {
               "$ref": "#/definitions/etcdserverpbAuthenticateResponse"
             }
+          },
+          "default": {
+            "description": "An unexpected error response",
```

Comment:
Nit, if you update commit for other changes, it would be good to keep the description consistent with rest of the file in this file and other swagger file by ending sentence with a period.

Reply:
Hmmm... this is actually auto generated by the genproto.sh script. Not sure why it is not consistent with the existing contents.

Reply:
Interesting! Not a big thing to worry about.